layouts/community/ortho_4x12/xyverz/rules.mk

```diff
@@ -1,4 +1,5 @@
 BACKLIGHT_ENABLE = yes      # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes       # Enable RGB Underglow functionality
```

Comment:
(Optional) Not sure if it matters here for you, but if you have only certain boards that you want this enabled for, you can do this: 

```makefile
ifdef KEYBOARD_lets_split_rev2
RGBLIGHT_ENABLE = no
else 
RGBLIGHT_ENABLE = yes
endif
```

Reply:
Actually, I have RGB Underglow on at least one of each of my four different 4x12 keebs, so this can definitely stay. I don't use the audio at all because those tiny piezo speakers don't quite have the power to be loud enough for this ol' half-deaf guy here. But thank you!

Reply:
Well, it can be for other features, :)

Just wanted to make sure that you're aware that you could, and how. :)
(in fact, I should add that to the documentation)